package.json

```diff
@@ -7,8 +7,9 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "serve": "http-server ./build",
-    "test": "react-scripts test",
+    "test": "npm run lint",
```

Comment:
When is `test` ran? I'm not sure I'm comfortable having eslint running with `--fix` automatically.

Reply:
@nerik  In carrier portal and data portal we run this task before the build image task in cloudbuild. Same in this project. Maybe the fix should be executed in a pre-commit instead of the test stage.

Reply:
>  Maybe the npm run lint with fix flag should be executed in a pre-commit instead of the test stage.

Personally I would prefer that, I think it's better to have human eyes reviewing linter changes (also an opportunity to learn)

Reply:
> > Maybe the npm run lint with fix flag should be executed in a pre-commit instead of the test stage.
> 
> Personally I would prefer that, I think it's better to have human eyes reviewing linter changes (also an opportunity to learn)

👍

Reply:
Thanks 👍

Reply:
Done @nerik